src/models/hooks/nationalCenter.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/prefer-default-export */
+const { Op } = require('sequelize');
+const { EVENT_REPORT_STATUSES } = require('@ttahub/common');
+
+const afterDestroy = async (sequelize, instance) => {
```

Comment:
Why is this an after destroy, and not an afterUpdate/afterCreate as all the work only needs to happen when there is a new national center that points to the old one.

Reply:
Updated the code with comments explaining what is going on